Documentation/gettingstarted/k8s-install-azure-cni-validate.rst

```diff
@@ -0,0 +1,50 @@
+Validate the Installation
+=========================
+
+You can monitor as Cilium and all required components are being installed:
+
+.. parsed-literal::
+
+    kubectl -n cilium get pods --watch
+    NAME                                    READY   STATUS              RESTARTS   AGE
+    cilium-operator-cb4578bc5-q52qk         0/1     Pending             0          8s
+    cilium-s8w5m                            0/1     PodInitializing     0          7s
+    coredns-86c58d9df4-4g7dd                0/1     ContainerCreating   0          8m57s
+    coredns-86c58d9df4-4l6b2                0/1     ContainerCreating   0          8m57s
```

Comment:
Remove the lines of coredns because coredns doesn't exist in the namespace cilium.

Reply:
Sure will address your comments in the next patch.

Reply:
@bingosummer Fixed review comments.